### ISSUE TITLE
Translations: plural forms warning

### ISSUE THREAD

**Issue by deveee** [supertuxkart/stk-code#1922]
I get this warning in terminal:
```
tinygettext: data/po//pl.po:18: warning: unknown Plural-Forms given:
tinygettext: data/po//pl.po:42: warning: msgstr[N] seen, but no Plural-Forms given:
```
In translations file you can see
```
#. I18N: Only used for count > 3
#: src/online/online_player_profile.cpp:407
#, c-format
msgid "%d friend is now online."
msgid_plural "%d friends are now online."
msgstr[0] "%d przyjaciel jest dostępny"
msgstr[1] "%d przyjaciół jest dostępnych"
msgstr[2] "%d przyjaciół jest dostępnych"
```
First warning is about header.

**Comment by hiker** [comment]
There was a missing () in the pl.po plural string. I don't know if pl.po is wrong, or the string to compare it again in stk. For now I've only fixed pl.po, this still needs to be done on launchpad I assume? Or in stk?

**Comment by deveee** [comment]
These warnings you can see also in other languages, eg. Russian.

On launchpad you probably won't do anything, because this header is automatically generated. Plural forms were implemented by someone other, but I will look if this can be easily fixed.

**Comment by deveee** [comment]
Should be fixed in ab01161c28c6be2a9149ed1cef2c8e57b7d38808.

**Comment by hiker** [comment]
Hi Deve,

thanks, I had no idea where those headers are created - I somehow
assumed you. Then it is indeed best to fix them in stk.

Cheers,
   Joerg
